### ISSUE TITLE
[next release] list of repos using doctr to PR fixes to

### ISSUE THREAD

**Issue by gforsyth** [drdoctr/doctr#160]
Following the merge and release of #157 

The list of repos I have using doctr (searching for `doctr` and filtering for YAML files) is:

* [ ] conda
* [ ] sympy
* [ ] xonsh (I can fix this)
* [ ] pygbe (I can fix this)
* [ ] essential_skills_RRC (I can fix this)
* [ ] https://github.com/pydata/pandas-design (although it says deprecated in the repo description)
* [ ] https://github.com/Carreau/warn
* [ ] https://github.com/pyoceans/sea-py
* [ ] https://github.com/maxpoint/spylon
* [ ] https://github.com/danielballan/photomosaic
* [ ] https://github.com/ergs/TWoFCS
* [ ] https://github.com/pystitch/stitch
* [ ] https://github.com/ergs/rg-db-twofcs
* [ ] https://github.com/python-visualization/folium
* [ ] https://github.com/sea-mat/sea-mat
* [ ] https://github.com/bryanwweber/UConnRCMPy
* [ ] https://github.com/bryanwweber/thermostate
* [ ] <strike>https://github.com/Alsheh/tmp</strike> sympy fork
* [ ] <strike>https://github.com/antepsis/anteplahmacun</strike> sympy fork

**Comment by bryanwweber** [comment]
Hi! I'm the @bryanwweber that's using Doctr for my two repositories (and I'm setting up a third right now). What do I need to do here?

**Comment by asmeurer** [comment]
Nothing until we do a release, but we are deprecating the `--gh-pages-docs` flag of `doctr deploy`, and replacing it with a requirement argument (like `doctr deploy .`). Once we do a release we want to make sure all these repos are updated so that the change will have minimal impact once we remove the flag entirely.

**Comment by bryanwweber** [comment]
OK got it. I'll watch for the release and update accordingly. Thanks!

**Comment by gforsyth** [comment]
Hey @bryanwweber -- the latest doctr has been released, so you should change your `doctr deploy` lines to just treat the deploy directory as a required argument. 
I'm happy to PR that in if you'd like.

**Comment by asmeurer** [comment]
Fixing SymPy at https://github.com/sympy/sympy/pull/12339

**Comment by gforsyth** [comment]
Don't. Critical bug

**Comment by gforsyth** [comment]
Something in the checks to only push from the master branch isn't working

**Comment by gforsyth** [comment]
The `push` travis run will push the docs

**Comment by asmeurer** [comment]
OK, will hold off.

**Comment by Carreau** [comment]
> Something in the checks to only push from the master branch isn't working

That might be in interference with the thing I did with branch whitelists. 
I'll dig into it as well.

**Comment by gforsyth** [comment]
I think I found it -- this line (https://github.com/drdoctr/doctr/blob/master/doctr/travis.py#L166)
`    if any([re.compile(x).match(TRAVIS_BRANCH) for x in branch_whitelist]):`
should be 
`    if not any([re.compile(x).match(TRAVIS_BRANCH) for x in branch_whitelist]):`
since we set `canpush=False` inside of that conditional. So instead we're allowing pushes from any branch except for `master` or anything in the whitelist

**Comment by Carreau** [comment]
Agreed, should we change `if branch_whitelist is None` to `if not branch_whitelist`, that won't behave the same if `branch_whitelist` is set to empty set or empty list.

**Comment by gforsyth** [comment]
ooh, yeah.  good point. I'll put that in, too.

**Comment by gforsyth** [comment]
Ok. 1.5.1 is out and on conda-forge. I've reopened the PRs I had to the repos listed above.

**Comment by bryanwweber** [comment]
@gforsyth Noted and I will update in the next commit I push to those repos
